content/guides/template-gds/_index.md

```diff
@@ -0,0 +1,227 @@
+   - For the Wokwi projects, the 'language' field should be set to 'Wokwi'.
+   - If your project has clock frequency specification, provide the frequency in Hz under the 'clock_hz' field.
+
+    <img src="images/wokwi_metadata.png" alt="WokwiMD" width="550" style="border: 1px solid #000;"/>
+3. **Describe Pins**:
+   - Provide descriptions for the pins to aid in testing the designs. 
+   - The Pin description are the inputs and outputs utilised in your Wokwi Project. You can refer the Wokwi project diagram to fill the pin description fields. 
+   - Note that the unused pins are to be left blank and additional pins cannot be included. 
+
+        <img src="images/info_yaml.png" alt="yamlinfo" width="100" style="border: 1px solid #000;"/>
+
+    - Save your changes by clicking the commit button.
+
+       <img src="images/commitinfo.png" alt="commitinfo" width="550" style="border: 1px solid #000;"/>
+<!-- 
+![info.yaml](images/info_yaml.png)  -->
+
+<!-- ![Commit the modified info.yaml](images/commitinfo.png) -->
+##  Edit the `info.md` File in the Docs Directory
+
+  **Update Documentation**:
+   - Include detailed instructions on how to experiment with your design. - This documentation is useful for users receiving the chip and for you to recall design details later and also generates the datasheet for your project. 
+   - Note that leaving the info.md unmodified will cause the 'docs' action to fail. 
+   - Save and commit your changes as you did before.
+
+      <img src="images/documentation.png" alt="doc" width="550" style="border: 1px solid #000;"/>
+<!-- ![Documentation](images/documentation.png) -->
+
+
+## Wokwi Actions
+
+1. **Access Actions Tab**:
+   - Go to the ‘Actions’ tab in your repository to view results. 
+   - Each commit will trigger these actions to run again. 
+   - Find out more: how the GitHub actions work to build the [GDS](https://www.zerotoasiccourse.com/terminology/gds2/) files for your project, you can check [here](https://tinytapeout.com/making_asics/).
+    
+2. **Actions for Wokwi**:
+
+    - __docs__: This action generates a preview of how your documentation will look.
+    - __fpga__: This action generates bitstream for a compatible FPGA
+    - __gds__: This action generates the gds for your project, results for warnings, utilization statistics, and cell usage.
+    - __wokwi test__: This action will test the design if you created a truth table for your wokwi design.
+    
+
+       <img src="images/actions.png" alt="actionslist" width="100" style="border: 1px solid #000;"/>
+
+    - Note that red indicates failure and green indicates success. Only the actions 'docs' and 'gds' need to be green.
+
+## Troubleshoot Action Failures 
+
+1. **Docs Action**:
+   - This action generates a preview of how your documentation will look.
+
+   - Check the action result to get the status of the check. 
+
+     <img src="images/CheckActions.png" alt="Docs Action Result" width="550" style="border: 1px solid #000;" />
```

Comment:
this and a few other images have been distorted because you are not keeping the aspect ratio fixed.